src/olympia/abuse/tests/test_views.py

```diff
@@ -147,9 +147,6 @@ def test_throttle(self):
         assert response.status_code == 429
 
     def test_optional_fields(self):
-        # FIXME: when choices for addon_signature, addon_install_method and
-        # addon_install_entry_point are finalized, update this test. In the
-        # meantime we pass None.
```

Comment:
it didn't need to be fixed?

Reply:
that's what this PR is about but I've added tests for the choices elsewhere (in `test_models` and `test_serializers`)